Makefile

```diff
@@ -1,4 +1,4 @@
-.PHONY: all binary calico/node test ut ut-circle st st-ssl clean run-etcd run-etcd-ssl help clean_calico_node
```

Comment:
I'd quite like to move the PHONY declarations to before the actual target - it's much easier to maintain.  Perhaps not one for this PR though.